wrappers/IWearLogger/src/IWearLogger.cpp

```diff
@@ -78,27 +107,42 @@ class IWearLogger::impl
         return res;
     }
 
-    inline std::string convertSensorNameToValidMatlabVarName(const std::string& sensorName)
+    inline std::string getValidName(std::string& name, const char& c)
```

Comment:
Is there a way to avoid changing the input `name` parameter? And I mean, making it const:

```suggestion
    inline std::string getValidName(const std::string& name, const char& c)
```

It's not very clean that this function permanently changes `name` that should be r/o.

Reply:
Done in https://github.com/robotology/wearables/pull/123/commits/afacfdf1c362f7bf87ef28f853f25037a6a3ca8a